src/Shared/Shared.csproj

```diff
@@ -33,6 +33,10 @@
     <PackageReference Include="System.Memory" Condition="!$([MSBuild]::IsTargetFrameworkCompatible('$(TargetFramework)', 'net6.0'))" />
   </ItemGroup>
 
+  <ItemGroup Condition="'$(TargetFrameworkIdentifier)' != '.NETCoreApp'">
```

Comment:
We also run the schema tests against .NET 9 to ensure the polyfill remains compatible with the in-box `JsonSchemaExporter`.